### PR TITLE
chore(*): update fondue version

### PR DESCRIPTION
**Pull request by fulopdaniel**
- Update fondue to latest
- Fix tests related to the update
- Remove outline from RTE

### CODE REVIEW

File: examples/asset-upload/tsconfig.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
         "jsx": "react-jsx",
         "lib": ["ES2021", "DOM"],
         "resolveJsonModule": true,
-        "types": ["cypress"],
+        "types": ["cypress", "cypress-real-events", "@4tw/cypress-drag-drop"],
```
Comment (ragi96) [inline examples/asset-upload/tsconfig.json:16]:
```suggestion
        "types": ["cypress"],
```

I don't think this change is needed as these packages are not installed on this example

File: packages/brand-positioning-block/tsconfig.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
         "jsx": "react-jsx",
         "lib": ["ES2021", "DOM"],
         "resolveJsonModule": true,
-        "types": ["cypress"],
+        "types": ["cypress", "cypress-real-events"],
```
Comment (ragi96) [inline packages/brand-positioning-block/tsconfig.json:16]:
```suggestion
        "types": ["cypress"],
```

Also here as `cypress-real-events` isn't used on that block

File: packages/callout-block/tsconfig.json [hunk 3]
```diff
@@ -13,7 +13,7 @@
         "jsx": "react-jsx",
         "lib": ["ES2021", "DOM"],
         "resolveJsonModule": true,
-        "types": ["cypress"],
+        "types": ["cypress", "cypress-real-events"],
```
Comment (ragi96) [inline packages/callout-block/tsconfig.json:16]:
```suggestion
        "types": ["cypress"],
```

Also this shouldn't be needed on that block

File: packages/color-kit-block/tsconfig.json [hunk 4]
```diff
@@ -13,7 +13,7 @@
         "jsx": "react-jsx",
         "lib": ["ES2021", "DOM"],
         "resolveJsonModule": true,
-        "types": ["cypress"],
+        "types": ["cypress", "cypress-real-events"],
```
Comment (ragi96) [inline packages/color-kit-block/tsconfig.json:16]:
```suggestion
        "types": ["cypress"],
```
Same here

File: packages/compare-slider-block/tsconfig.json [hunk 5]
```diff
@@ -12,6 +12,7 @@
         "target": "es6",
         "jsx": "react-jsx",
         "lib": ["ES2021", "DOM"],
+        "types": ["cypress", "cypress-real-events", "@4tw/cypress-drag-drop"],
```
Comment (ragi96) [inline packages/compare-slider-block/tsconfig.json:15]:
```suggestion
        "types": ["cypress"],
```
Same here

File: packages/divider-block/tsconfig.json [hunk 6]
```diff
@@ -13,7 +13,7 @@
         "jsx": "react-jsx",
         "lib": ["ES2021", "DOM"],
         "resolveJsonModule": true,
-        "types": ["cypress"],
+        "types": ["cypress", "cypress-real-events"],
```
Comment (ragi96) [inline packages/divider-block/tsconfig.json:16]:
```suggestion
        "types": ["cypress"],
```
Same here

File: packages/dos-donts-block/tsconfig.json [hunk 7]
```diff
@@ -13,7 +13,7 @@
         "jsx": "react-jsx",
         "lib": ["ES2021", "DOM"],
         "resolveJsonModule": true,
-        "types": ["cypress"],
+        "types": ["cypress", "cypress-real-events"],
```
Comment (ragi96) [inline packages/dos-donts-block/tsconfig.json:16]:
```suggestion
        "types": ["cypress"],
```
Same

File: packages/example/tsconfig.json [hunk 8]
```diff
@@ -13,7 +13,7 @@
         "jsx": "react-jsx",
         "lib": ["ES2021", "DOM"],
         "resolveJsonModule": true,
-        "types": ["cypress"],
+        "types": ["cypress", "cypress-real-events"],
```
Comment (ragi96) [inline packages/example/tsconfig.json:16]:
```suggestion
        "types": ["cypress"],
```
Same here

File: packages/figma-block/tsconfig.json [hunk 9]
```diff
@@ -13,7 +13,7 @@
         "jsx": "react-jsx",
         "lib": ["ES2021", "DOM"],
         "resolveJsonModule": true,
-        "types": ["cypress"],
+        "types": ["cypress", "cypress-real-events"],
```
Comment (ragi96) [inline packages/figma-block/tsconfig.json:16]:
```suggestion
        "types": ["cypress"],
```

File: packages/image-block/tsconfig.json [hunk 10]
```diff
@@ -13,7 +13,7 @@
         "jsx": "react-jsx",
         "lib": ["ES2021", "DOM"],
         "resolveJsonModule": true,
-        "types": ["cypress"],
+        "types": ["cypress", "cypress-real-events"],
```
Comment (ragi96) [inline packages/image-block/tsconfig.json:16]:
```suggestion
        "types": ["cypress"],
```

File: packages/personal-note-block/tsconfig.json [hunk 11]
```diff
@@ -13,7 +13,7 @@
         "jsx": "react-jsx",
         "lib": ["ES2021", "DOM"],
         "resolveJsonModule": true,
-        "types": ["cypress"],
+        "types": ["cypress", "cypress-real-events"],
```
Comment (ragi96) [inline packages/personal-note-block/tsconfig.json:16]:
```suggestion
        "types": ["cypress"],
```

File: packages/quote-block/tsconfig.json [hunk 12]
```diff
@@ -13,7 +13,7 @@
         "jsx": "react-jsx",
         "lib": ["ES2021", "DOM"],
         "resolveJsonModule": true,
-        "types": ["cypress"],
+        "types": ["cypress", "cypress-real-events"],
```
Comment (ragi96) [inline packages/quote-block/tsconfig.json:16]:
```suggestion
        "types": ["cypress"],
```

File: packages/shared/package.json [hunk 13]
```diff
@@ -47,17 +47,17 @@
         "@dnd-kit/modifiers": "6.0.1",
```
Comment (ragi96) [inline packages/shared/package.json:47]: Also update `slate` and `slate-react` within fondue

File: packages/shared/src/components/Attachments/Attachments.spec.ct.tsx [hunk 14]
```diff
@@ -125,11 +125,14 @@ describe('Attachments', () => {
...
-            .trigger('keydown', { key: 'Tab' })
-            .trigger('keydown', { key: 'Tab' });
+        cy.realPress('Tab');
+        cy.realPress('Tab');
+        cy.realPress('Tab');
+        // cy.get(AttachmentItemSelector)
+        //     .eq(0)
+        //     .trigger('keydown', { key: 'Tab' })
+        //     .trigger('keydown', { key: 'Tab' })
+        //     .trigger('keydown', { key: 'Tab' });
```
Comment (ragi96) [inline packages/shared/src/components/Attachments/Attachments.spec.ct.tsx:135]:
```suggestion
```

remove commented code

File: packages/shared/src/components/RichTextEditor/RichTextEditor.tsx [hunk 15]
```diff
@@ -40,16 +40,18 @@ export const RichTextEditor = ({
...
-                id={id}
-                value={value}
-                border={false}
-                placeholder={placeholder}
-                plugins={plugins}
-                updateValueOnChange={updateValueOnChange}
-                onValueChanged={() => setShouldPreventPageLeave(true)}
-                onTextChange={saveText}
-            />
+            <div className="[&_[contenteditable]]:tw-outline-none">
```
Comment (ragi96) [inline packages/shared/src/components/RichTextEditor/RichTextEditor.tsx:43]: Just wondering why this div is needed?

File: packages/sketchfab-block/tsconfig.json [hunk 16]
```diff
@@ -13,7 +13,7 @@
         "jsx": "react-jsx",
         "lib": ["ES2021", "DOM"],
         "resolveJsonModule": true,
-        "types": ["cypress"],
+        "types": ["cypress", "cypress-real-events"],
```
Comment (ragi96) [inline packages/sketchfab-block/tsconfig.json:16]:
```suggestion
        "types": ["cypress"],
```

not needed

File: packages/storybook-block/tsconfig.json [hunk 17]
```diff
@@ -13,7 +13,7 @@
         "jsx": "react-jsx",
         "lib": ["ES2021", "DOM"],
         "resolveJsonModule": true,
-        "types": ["cypress"],
+        "types": ["cypress", "cypress-real-events"],
```
Comment (ragi96) [inline packages/storybook-block/tsconfig.json:16]:
```suggestion
        "types": ["cypress"],
```

File: packages/text-block/tsconfig.json [hunk 18]
```diff
@@ -13,7 +13,7 @@
         "jsx": "react-jsx",
         "lib": ["ES2021", "DOM"],
         "resolveJsonModule": true,
-        "types": ["cypress"],
+        "types": ["cypress", "cypress-real-events"],
```
Comment (ragi96) [inline packages/text-block/tsconfig.json:16]:
```suggestion
        "types": ["cypress"],
```

**Review by ragi96 — CHANGES_REQUESTED** [review]
Not working, just empty text block
Screenshot 2023-07-04 at 09 24 48

But its working with an empty audio block

Can have multiple toolbars (should be just one)
Screenshot 2023-07-04 at 09 27 34

Toolbar open after page change still there
Screenshot 2023-07-04 at 09 28 54

File: packages/shared/src/components/RichTextEditor/RichTextEditor.tsx (hunk 15 above)
Comment (fulopdaniel) [reply packages/shared/src/components/RichTextEditor/RichTextEditor.tsx:43]: New RTE adds a blue outline which was not there previously.

File: packages/shared/src/components/Attachments/Attachments.spec.ct.tsx (hunk 14 above)
Comment (fulopdaniel) [reply packages/shared/src/components/Attachments/Attachments.spec.ct.tsx]: Thanks, done

File: packages/shared/src/components/RichTextEditor/RichTextEditor.tsx [hunk 19]
```diff
@@ -5,6 +5,7 @@ import React, { useEffect, useState } from 'react';
 import { RichTextEditor as FondueRichTextEditor } from '@frontify/fondue';
```
Comment (ragi96) [inline packages/shared/src/components/RichTextEditor/RichTextEditor.tsx:5]: Button Working, Link not

https://www.loom.com/share/202a5eb8ff2e4788b46e2f6c3ada30b5?sid=47212c60-16ef-48d5-a1ca-5bfade2d8a0b

**Review by ragi96 — COMMENTED** [review]
Also blinking placeholder for thumbnail grid block, animation curve block

https://www.loom.com/share/b768dc8a424743a08a374b309585f622?sid=ec99273e-4c74-4325-98f2-ba58084aab35

File: packages/shared/src/components/RichTextEditor/RichTextEditor.tsx (hunk 19 above)
Comment (fulopdaniel) [reply packages/shared/src/components/RichTextEditor/RichTextEditor.tsx:5]: i could not reproduce this with the new zIndices. Can you check again?

File: packages/audio-block/src/AudioBlock.tsx [hunk 20]
```diff
@@ -131,7 +131,7 @@ export const AudioBlock = ({ appBridge }: BlockProps) => {
                     />
                 )
             )}
-            <div className="tw-flex tw-gap-4 tw-justify-between tw-w-full tw-relative tw-z-20">
+            <div className="tw-flex tw-gap-4 tw-justify-between tw-w-full tw-relative">
```
[the remark this answers is not in the document]
Comment (fulopdaniel) [reply packages/audio-block/src/AudioBlock.tsx:131]: Should be fixed!

File: examples/asset-upload/package.json [hunk 21]
```diff
@@ -31,7 +31,7 @@
     },
     "dependencies": {
         "@frontify/app-bridge": "3.0.0-beta.67",
-        "@frontify/fondue": "12.0.0-beta.239",
+        "@frontify/fondue": "v12.0.0-beta.279",
```
[the remark this answers is not in the document]
Comment (fulopdaniel) [reply examples/asset-upload/package.json]: My bad! fixed

File: pnpm-lock.yaml
Comment (ragi96) [inline pnpm-lock.yaml:1]: Can you check it, as it seems to have to many changes

**Pull request merged**